magicbot/magic_tunable.py

```diff
@@ -137,6 +157,13 @@ def setup_tunables(component, cname: str, prefix: Optional[str] = "components")
         )
         tunables[prop] = ntvalue
 
+        if prop._update_cb:
+            prop._nt._api.addEntryListenerById(
+                ntvalue._local_id,
+                functools.partial(prop._update_cb, component),
```

Comment:
Using a lamdba above and a partial here? Why the mix?

Reply:
If we use a lambda here, we'd have to pass the component in as a default-arg. Can't use a closure since it's in a loop.

Reply:
`functools.partial` is implemented in C? :man_shrugging:

Reply:
Actually I only just realised they can be consolidated to a single lambda here.